src/main/java/de/cerus/jdasc/JDASlashCommands.java

```diff
@@ -552,7 +585,7 @@ public static void handleInteraction(final Interaction interaction) {
      *
      * @return A argument or null
      */
-    private static Map<String, InteractionResponseOption> findArguments(final ApplicationCommand command, final Interaction interaction) {
+    public static Map<String, InteractionResponseOption> findArguments(final ApplicationCommand command, final Interaction interaction) {
```

Comment:
Why did you make this public? Maybe I'm missing something, but this doesn't seem to have a purpose

Reply:
Wanted to first implement a custom event to get args, but realized its easier doing it the current way